tests/utils.py

```diff
@@ -52,3 +54,8 @@
     # Associate the peer with local ip address (see default parameters of Libp2p())
     node_a.get_peerstore().add_addrs(node_b.get_id(), node_b.get_addrs(), 10)
     return node_a, node_b
+
+
+def generate_new_private_key():
+    new_key = RSA.generate(2048, e=65537)
+    return new_key
```

Comment:
looks identical to the `libp2p.generate_new_private_key` function.  Any reason it's duplicated?

Reply:
it is :)

"a little copy is better than a little dependency", esp when the code is short and you are moving fast ;)

i'll make a note to deprecate this (we need a better way to handle key gen anyway)

Reply:
also fixed in other work, reflected in #250.